src/main/java/com/Incubatormsrv/Incubatormsrv/weather/mapper/WeatherMapper.java

```diff
@@ -0,0 +1,15 @@
+package com.Incubatormsrv.Incubatormsrv.weather.mapper;
+
+import _Users_luyanda_glucode.com_Documents_GitHub_Incubator_msrv.model.HelloWorld;
+import _Users_luyanda_glucode.com_Documents_GitHub_Incubator_msrv.model.WeatherResponse;
+import org.springframework.stereotype.Component;
+
+@Component
+public class WeatherMapper {
+
+    public HelloWorld helloWorldMapper(WeatherResponse weatherResponse) {
```

Comment:
Similarly confused with the names and purposes here. Looks like you're shoehorning the waether data into the HelloWorld response.
Please generate an appropriate response object for the weather data and map to that object.

Reply:
weatherResponse in this case is my weatherApi Model, which I am mapping with hello world.